gluoncv/loss.py

```diff
@@ -294,24 +295,53 @@ class SoftmaxCrossEntropyLossWithAux(SoftmaxCrossEntropyLoss):
     ignore_label : int, default -1
         The label to ignore.
     """
-    def __init__(self, aux=True, aux_weight=0.2, ignore_label=-1, **kwargs):
+    def __init__(self, aux=True, mixup=False, aux_weight=0.2, ignore_label=-1, **kwargs):
```

Comment:
Propose to change the name of the class, since 'Aux' means two losses, while mixup means two labels. 

Also preferrably incorporate label smoothing into the loss: https://github.com/hetong007/gluon-cv/blob/master/scripts/classification/imagenet/train_imagenet.py#L266